os/arch/arm/src/armv7-m/up_systick.c

```diff
@@ -0,0 +1,300 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
```

Comment:
2019

Reply:
NXP (imxrt chip) does not use this file. Why do you add this?

Reply:
I will remove it.